UnitySDK/Assets/ML-Agents/Scripts/InferenceBrain/TensorProxy.cs

```diff
@@ -90,6 +90,23 @@ public static TensorProxy TensorProxyFromBarracuda(Tensor src, string nameOverri
             };
         }
 
+        public static void FillTensorBatch(TensorProxy tensorProxy, int batch, float fillValue)
```

Comment:
Could do this as a method on TensorProxy, but it follows the convention in FillTensorWithRandomNormal (below)